src/main/scala/com/sksamuel/scapegoat/package.scala

```diff
@@ -1,8 +1,9 @@
 package com.sksamuel
 
 package object scapegoat {
-  val scalaVersion = util.Properties.versionNumberString
-  val shortScalaVersion = scalaVersion.split('.').dropRight(1).mkString(".")
+  val scalaVersion: String = util.Properties.versionNumberString
+  private val shortScalaVersion = scalaVersion.split('.').dropRight(1).mkString(".")
 
-  val isScala213 = shortScalaVersion == "2.13"
+  val isScala213: Boolean = shortScalaVersion == "2.13"
+  val isScala21312: Boolean = scalaVersion == "2.13.12"
```

Comment:
what if instead of string comparison, we'll try to use major/minor/patch versions as numbers?
Something like this, but ensuring that the split is 'safe':
```scala
val (major, minor, patch) = scalaVersion.split('.').map(_.toInt)
val isScala213: Boolean = major == 2 && minor == 13
val isScala21312OrLater: Boolean = isScala213 && patch >= 12
```

Reply:
Good idea! We'd welcome your PR with this small change.

Reply:
Looks to become a gotcha for new 2.13 versions. Probably can find some time this week to raise this as an MR.

How lenient do we want to be here when we can't parse the string?
- Fail fast (as scapegoat is full cross-built, might not be the worst)
- Fail per component (2.13.231-snapshot could still be understood as 2.13.??? so the is 2.13 check would still work)
- Fallback to something (not sure there's a sane default considering 2.12/2.13 though)

Reply:
I still think what @saeltz mentioned here would be a good amendment.

Reply:
https://github.com/scapegoat-scala/scapegoat/pull/826 raised here :)